cross/libvpx/digests

```diff
@@ -0,0 +1,3 @@
+v1.5.0.tar.gz SHA1 82426a59ad34d72c762f9e4079e8e5197afc842d
```

Comment:
Recreate digests please. This should be based off of PKG_DIST_FILE, not PKG_DIST_NAME